### PR TITLE
Minor tweaks to work with SecrectClient updates

### PR DESCRIPTION
**Pull request by AnthonyMBonafide**
Updated configuration toml and keys used to obtain database secrets

Signed-off-by: Anthony M. Bonafide <AnthonyMBonafide@gmail.com>

### CODE REVIEW

**Comment by AnthonyMBonafide** [comment]
After doing some manual acceptance tests, I found some issues that came up due to last minute cleanup changes I made with the client. Let me know if there are some changes that can be made to the client to make user's interaction more ergonomic.

File: internal/core/data/init.go [hunk 1]
```diff
@@ -231,12 +232,12 @@ func connectAndPollSecrets() error {
 		return err
 	}
 
-	username, err := secretsClient.GetSecret("coredata")
+	username, err := secretsClient.GetSecret("User")
```
Comment (tsconn23) [inline internal/core/data/init.go]: Should `User` be a constant exposed from the SecretClient?

File: internal/core/data/init.go [hunk 2]
```diff
@@ -231,12 +232,12 @@ func connectAndPollSecrets() error {
...
 	if err != nil {
 		return err
 	}
 
-	password, err := secretsClient.GetSecret("coredatapasswd")
+	password, err := secretsClient.GetSecret("Passwd")
```
Comment (tsconn23) [inline internal/core/data/init.go]: Should `Passwd` be a constant exposed from the SecretClient? (spell it out please)

Comment (AnthonyMBonafide) [reply internal/core/data/init.go]: > Should `Passwd` be a constant exposed from the SecretClient? (spell it out please -- Password)

Good suggestion. @tingyuz, is there something I need to do to update the key that was stored in the Vault instance I was testing the client against? Currently the key is `Passwd` for your test instance.

**Pull request merged**